packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -52,6 +52,7 @@ export const messages = {
   parameter_gatherer_enter_file_name: 'Enter desired filename',
   parameter_gatherer_enter_dir_name:
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
+  parameter_gatherer_enter_lwc_name: 'Enter desired Lightning Web Component(Press Enter to confirm or Esc to cancel)',
```

Comment:
```suggestion
  parameter_gatherer_enter_lwc_name: 'Enter desired Lightning Web Component (Press Enter to confirm or Esc to cancel)',
```

Can you upload a screenshot of where the message surfaces as a reference for @ekapner and @hbc-bindu?

Reply:
@brpowell - I poked around in VS Code using all the Lightning commands and I couldn't get this to pop up. And I couldn't find any lightning localization commands where this might pop up either. We might need to reach out to LWC doc team to see if they know. Any way to tell to which command this is associated?

Reply:
@ekapner Were you poking around with Amanda's branch? This is a brand new command being added so it isn't in the wild yet. We'll need to add the command to the doc site as a part of this PR as well.

Reply:
@brpowell - I thought you were asking us in doc for the screen shot - and that's when I went searching for it. Sorry, I read your comment wrong. But based on the other Lightning commands, the highlighted messages requires a very small edit (to make them consistent).

Press 'Enter' to confirm or 'Escape' to cancel

Reply:
In short, please provide the screen shot, and @hbc-bindu will update the docs.

Reply:
Ah whoops, I meant @amandastern99 add a screenshot for you to reference :)

Reply:
<img width="593" alt="Screen Shot 2020-06-02 at 4 22 22 PM" src="https://user-images.githubusercontent.com/35906675/83565959-403a7a00-a4ed-11ea-8d13-24c0e4510819.png">


Here is what it looks like!

Reply:
FYI @hbc-bindu - do you need anything else?